rxjava/build.gradle

```diff
@@ -9,14 +9,15 @@ android {
     testBuildType "qa"
 
     defaultConfig {
-
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
-
     }
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
+        release.java.srcDirs += 'src/main/kotlin'
```

Comment:
This is not related to this specific PR, but we should make this coherent across Skeletoid...